### PR TITLE
Added employee page + update github instructions

### PR DESCRIPTION
**Pull request by morecallan**
- Still need to make a decision on the .gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,47 +12,61 @@ Pick a theme that the team likes. A good place for inspiration is the [Wordpress
...
 1. Company awards
 
-Regardless of what pages each team member works on, each team must have a Products page for their site that the team will collaborate on. See more details about the product page below.
+Regardless of what pages each team member works on, each team must have a Products page and an Employee page for their site that the team will collaborate on. See more details about the product page below.
 
 ### Git
 Your team is going to learn how to use git to enable good teamwork when building a software product.
 
-1. Now that everyone has joined the same team, everyone clone the Github Classroom project into your current directory - which should be `group-projects` if you ran the setup commands.
+1. Now that everyone has joined the same team, everyone clone the Github Classroom project into your current directory - which should be `groupprojects` if you ran the setup commands.
```
Comment (zoeames) [inline README.md]: group_projects

File: README.md [hunk 2]
```diff
@@ -20,47 +12,61 @@ Pick a theme that the team likes. A good place for inspiration is the [Wordpress
...
     1. `git commit -m "Initial commit with .gitignore file"`
     1. `git push origin YOUR BRANCH NAME`
-1. Now make a PR and get your team to approve it and merge into master.
-1. Now everyone should `git pull origin master`.
+1. Now make a PR and get your team to approve it :+1: and merge into master on GitHub.
+1. Now everyone should `git pull origin master` (from their local master branch).
 1. That's the last commit you make on the master branch. At this point, each teammate will create their branch.
 
-Now everyone will work on their page on those individual branches. Only when teammates feel that the work is complete, the teammate will push up their branch to Github and create a pull request. One of the other teammates must review the pull request and give a thumbs up (Just type :+1 in the comment box).
+Now everyone will work on their page on those individual branches. Once a teammates feels that their work is complete, the teammate will push up their branch to Github and create a pull request. One of the other teammates must review the pull request and give a thumbs up (Just type :+1 in the comment box).
```
Comment (zoeames) [inline README.md]: maybe elaborate a little more on reviewing PRs - look over the code on the PR.  pull the code down and run it to make sure it works.

**Comment by zoeames** [comment]
👍

**Comment by rousell** [comment]
Add this before the ###Git instructions

### Teamwork
Create a Project board for your company site and make the columns for ('backlog', 'doing', 'blocked', 'in review', 'done'). Create notes for different units of work, make the notes into issues, and assign it to the team members doing to work. When you are assigned to an issue, make sure that you are moving it over to the right column on the board.

**Comment by rousell** [comment]
^^ @morecallan

**Comment by rousell** [comment]
👍

**Comment by rousell** [comment]
@morecallan should we merge this in?

**Comment by zoeames** [comment]
lol yes

**Pull request merged**